draft/2022-04-27-this-week-in-rust.md

```diff
@@ -29,24 +29,42 @@ and just ask the editors to select the category.
 
 ### Official
 
+* [Rust Library Team Aspirations](https://blog.rust-lang.org/inside-rust/2022/04/20/libs-aspirations.html)
+* [video] [Cross Team Collaboration Fun Times (CTCFT)](https://www.youtube.com/watch?v=xOkI7xZ35fE)
+
 ### Foundation
 
 ### Newsletters
 
 ### Project/Tooling Updates
 
+* [rust-analyzer changelog #126](https://rust-analyzer.github.io/thisweek/2022/04/25/changelog-126.html)
+* [IntelliJ Rust Changelog #169](https://intellij-rust.github.io/2022/04/25/changelog-169.html)
 * [Slint (UI crate) weekly update](https://slint-ui.com/thisweek/2022-04-25.html)
+* [Hello, Robyn!](https://www.sanskar.me/hello_robyn.html)
+* [gitoxide - Rich repository information & blazingly fast clone-checkouts](https://github.com/Byron/gitoxide/discussions/398)
 
 ### Observations/Thoughts
 
+* [A little fixed point math for embedded audio](https://jamesmunns.com/blog/fixed-point-math/)
 * [Indirect ownership, shallow borrow and self-referential data structures](https://yoyo-code.com/indirect-ownership-and-self-borrow/)
+* [Rust traits and dependency injection](https://jmmv.dev/2022/04/rust-traits-and-dependency-injection.html)
+* [What is raw identifier in rust?](https://inspektor.cloud/blog/raw-identifier-in-rust/)
+* [How hard upgrading a Rust JWT library could be?](https://blog.orhun.dev/upgrading-rust-jwt)
```

Comment:
Looks like a 301 redirect to https://blog.orhun.dev/upgrading-rust-jwt/ (with trailing slash)

Reply:
I remembered that I can actually push commits to PR branches, so I just fixed that URL myself :)